src/v0/destinations/canny/util.js

```diff
@@ -40,16 +44,17 @@ const retrieveUserId = async (apiKey, message) => {
     response = await httpPOST(url, qs.stringify(requestBody), header);
     // If the request fails, throwing error.
     if (response.success === false) {
-      throw new CustomError(
-        `[Canny]:: CannyUserID can't be gnerated due to ${response.data.error}`,
-        400
+      throw new NetworkInstrumentationError(
```

Comment:
How can we deterministically call it an instrumentation error?

Reply:
Here, the request is failing but we are not sure about the reason. But we are taking all the fields for the request body from the user only. What should be the suitable error category for this?

Reply:
This should be safely attributed to NetworkError in order to fwd details to the caller service

Reply:
Yes.